### DISCUSSION TITLE
How to install on Ubuntu 22.04?

### DISCUSSION THREAD

**Discussion by WrichikBasu** [Keruspe/GPaste#405] [Q&A]
The Ubuntu package repository has an outdated version of Gpaste: [v3.42.6-1](https://packages.ubuntu.com/jammy/gpaste). The latest version in this repo is 42.1 at the time of writing this.

How do I install the latest version on Ubuntu Jammy? No `.deb` file is provided with the release.

**Comment by madeeldevops** [comment]
have you install it or not, i am also stuck. And would love to have a manual on it

**Reply by WrichikBasu** [reply]
Nope, I couldn't install and ditched it. I am now using [Clipboard History](https://extensions.gnome.org/extension/4839/clipboard-history/), which works fine.

**Comment by Keruspe** [comment]
Not sure about 22.04 but for recent versions of Ubuntu it's: sudo apt install gpaste gnome-shell-extension-gpaste
